### DISCUSSION TITLE
docker run mongo failed

### DISCUSSION THREAD

**Discussion by JordanRD** [docker-library/mongo#665] [General]
i use windows 

<img width="756" alt="Screenshot 2024-01-19 165412" src="https://github.com/docker-library/mongo/assets/73149617/a4afea09-467b-47ba-b762-6d552b572dda">
<img width="747" alt="Screenshot 2024-01-19 165440" src="https://github.com/docker-library/mongo/assets/73149617/e8b264e3-f8e6-409c-a451-a354cf90998b">

**Comment by tianon** [comment]
![image](https://github.com/docker-library/mongo/assets/161631/10972add-469a-4272-b411-0a4966d8ece7)

I think this is your best clue (in the future, you should copy/paste the text from the terminal instead of screenshotting, though)
